plugins/modules/keycloak_user.py

```diff
@@ -101,6 +101,8 @@
   groups:
     description:
       - List of groups for the user.
+        Groups can be referenced by their name, like "staff", or their path, like "/staff/engineering".
+        The path syntax allows you to reference subgroups, which is not possible otherwise.
```

Comment:
```suggestion
        The path syntax allows you to reference subgroups, which is not possible otherwise.
        This is possible since community.general 10.6.0.
```